src/pool.py

```diff
@@ -0,0 +1,18 @@
+from typing import  TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from .client import DiscordClient
+
+class pool:
+    
+    def __init__(self, tokens: list[str]) -> None:
+        self.tokens: list[str] = tokens
```

Comment:
```suggestion
        self.tokens = tokens
```
`tokens` is already typed as a list of strings so self.tokens will he typed as that I believe

Reply:
Yup fixed that